src/data_access/API/GOVehiclePositionApiClass.java

```diff
@@ -0,0 +1,59 @@
+package data_access.API;
+
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GOVehiclePositionApiClass implements TrainApiInterface {
+
+    private final String PARTIAL_API_URL = "OpenDataAPI/api/V1";
+    public GOVehiclePositionApiClass () {
+    }
+    public List<String> retrieveVehiclePosition(String stationId){
+        OkHttpClient client = new OkHttpClient().newBuilder()
+                .build();
+        HttpUrl httpUrl = new HttpUrl.Builder()
+                .scheme("http")
+                .host("api.openmetrolinx.com")
+                .addPathSegment(PARTIAL_API_URL)
+                .addPathSegment("UP")
```

Comment:
UP/Gtfs/Feed
This pulls UP Express info. We're doing regular GO vehicles

Reply:
## Future reference:

TraIns
http://api.openmetrolinx.com/OpenDataAPI/api/V1/ServiceataGlance/Trains/All?key=[API_KEY]
Buses
http://api.openmetrolinx.com/OpenDataAPI/api/V1/Gtfs/Feed/TripUpdates?key=[API_KEY]